core/src/modules/configLoaders/source.ts

```diff
@@ -101,7 +111,11 @@ export async function loadSource(
   }
 
   logModel(source, validate ? "validated" : isNew ? "created" : "updated");
-  if (bootstrappedProperty) {
+  if (configObject.bootstrappedProperty) {
+    log(
+      `source.bootstrappedProperty is deprecated. You can generate the property by using \`grouparoo generate\` as usual and the bootstrapped property will be automatically determined. (${configObject.id})`,
```

Comment:
👍 for the warning message. 

I don't think it will "automatically determined" though, unless you do the `--with` ?  How about:

> source.bootstrappedProperty is deprecated. Please generate a the config file for the Property by using \`grouparoo generate\`. (${configObject.id})